data_transfer/generic_https_transfer.py

```diff
@@ -38,10 +40,10 @@ def main(
         raise Exception(f"Incorrect URLs: {incorrect_urls}")
 
     sb = hb.ServiceBackend(
-        billing_project=os.getenv("HAIL_BILLING_PROJECT"),
+        billing_project=billing_project,
         remote_tmpdir=remote_tmpdir(),
     )
-    batch = hb.Batch(f"transfer {DATASET}", backend=sb, default_image=DRIVER_IMAGE)
+    batch = hb.Batch(f"transfer {DATASET}", backend=sb, default_image=cpg_driver_image)
 
     output_path = f"gs://cpg-{DATASET}-main-upload"
```

Comment:
Should we update this line to be `dataset_path(subfolder, 'upload')`?

Reply:
Thanks, I was just going to check that one. I think we have to... `Dataset` is also in that config file instead of the env

Reply:
ok, pushed another commit. The intention is to provide a sample_id or similar as the `--output` argument

Neither dataset_path or output_path were a great fit for this... output_path requires a suffix on top of the `--output` value